en/api_reference/classdronecore_1_1_drone_core.md

```diff
@@ -28,17 +27,17 @@ Type | Name | Description
 ---: | --- | ---
 | [DroneCore](#classdronecore_1_1_drone_core_1a0b94dd09cd46faa41742d3720f210aa2) () | Constructor.
```

Comment:
Yes and no.

In gitbook (our actual rendered library) the syntax is used to define a heading anchor. Github uses a slightly different flavour of markup which doesn't render this properly. While it would be good for this to render OK in both cases, getting it right in gitbook is more important.